webpack.config.js

```diff
@@ -1,34 +1,40 @@
-const UglifyJsPlugin = require('uglifyjs-webpack-plugin')
-const path = require('path')
-const TsConfigPathsPlugin = require('tsconfig-paths-webpack-plugin')
-const nodeExternals = require('webpack-node-externals');
```

Comment:
Why remove this?

Reply:
@anonymoussprocket  I used the webpack to compile the conseiljs.min.js.
We need to embed the min file using a script tag on an HTML file.
There is no node_module.
So we need to import node modules when compile.